src/main/java/gregtech/api/unification/material/materials/OrganicChemistryMaterials.java

```diff
@@ -470,5 +470,32 @@ public static void register() {
                 .flags(GENERATE_GEAR, GENERATE_RING, FLAMMABLE, NO_SMASHING, DISABLE_DECOMPOSITION, GENERATE_FOIL, GENERATE_BOLT_SCREW)
                 .components(Carbon, 5, Hydrogen, 8)
                 .build();
+
+        Cyclohexane = new Material.Builder(1069, "cyclohexane")
+                .fluid()
+                .color(0xF2F2F2E7)
+                .components(Carbon, 6, Hydrogen, 12)
+                .build();
+
+        NitrosylChloride = new Material.Builder(1070, "nitrosyl_chloride")
+                .fluid(Material.FluidType.GAS)
+                .flags(FLAMMABLE)
+                .color(0xF3F100)
+                .components(Nitrogen, 1, Oxygen, 1, Chlorine, 1)
+                .build();
+
+        Cyclohexanoxime = new Material.Builder(1071, "cyclohexanoxime")
```

Comment:
The variable, unlocalized name, and localized name should be renamed to `CyclohexanoneOxime`, `cyclohexanone_oxime`, and `Cyclohexanone Oxime` respectively.